inc/savedsearch.class.php

```diff
@@ -285,7 +285,7 @@ function prepareInputForAdd($input) {
          return false;
       }
 
-      $taburl = parse_url(rawurldecode($input['url']));
```

Comment:
We miss the sanitizer part from 10 here.
But i don't know what should be used, `Toolbox::stripslashes_deep` ?

@cedric-anne

Reply:
`Toolbox::unclean_cross_side_scripting_deep(Toolbox::stripslashes_deep($input['url']))` will both handle slashes and html special chars encoding.